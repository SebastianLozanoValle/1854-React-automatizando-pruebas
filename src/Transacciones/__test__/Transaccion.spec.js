import Transaccion from "../Transaccion";
import { render } from "@testing-library/react";

describe("Componente transaccion", () => {
  it("Genera un snapshot y tiene la misma estructura", () => {
    const { container } = render(
      <Transaccion data="25/09/2021" tipo="retiro" valor="200" />
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
