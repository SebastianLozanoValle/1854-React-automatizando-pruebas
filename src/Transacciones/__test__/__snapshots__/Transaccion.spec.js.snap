// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente transaccion Genera un snapshot y tiene la misma estructura 1`] = `
<div
  class="container"
>
  <p>
    25/09/2021
  </p>
  <p>
    RETIRO
  </p>
  <p>
    $ 
    200
  </p>
</div>
`;
